### PR TITLE
Add error handling for Deployments, LoadBalancer extension

### PR DESCRIPTION
**Pull request by drshrey**

### CODE REVIEW

**Comment by drshrey** [comment]
addresses https://github.com/codeamp/circuit/issues/103

**Comment by drshrey** [comment]
also addresses https://github.com/codeamp/circuit/issues/102

File: plugins/kubernetes/deployments/deployments.go [hunk 1]
```diff
@@ -64,7 +65,15 @@ func (x *Deployments) Process(e transistor.Event) error {
 	event.State = plugins.GetState("complete")
 	event.StateMessage = "Completed"
 
-	x.doDeploy(e)
+	err := x.doDeploy(e)
+	spew.Dump(err)
```
Comment (amoskyler) [inline plugins/kubernetes/deployments/deployments.go]: we got spew stuff.

Comment (drshrey) [reply plugins/kubernetes/deployments/deployments.go]: ah thanks, removing

File: plugins/kubernetes/deployments/util.go [hunk 2]
```diff
@@ -420,7 +423,7 @@ func (x *Deployments) doDeploy(e transistor.Event) error {
 	// Validate we have some services to deploy
 	if len(reData.Release.Services) == 0 {
 		x.sendDDErrorResponse(e, reData.Release.Services, "ERROR: Zero services were found in the deploy message.")
-		return nil
+		return fmt.Errorf("ERROR: Zero services were found in the deploy message.")
```
Comment (amoskyler) [inline plugins/kubernetes/deployments/util.go]: we use the same string twice - once on 425 and another time here. should write it into a variable first

**Pull request merged**